common/sketch/src/main/java/org/apache/spark/util/sketch/CountMinSketch.java

```diff
@@ -124,6 +124,18 @@ int getVersionNumber() {
    */
   public abstract void add(Object item, long count);
 
+  public abstract void addLong(long item);
```

Comment:
need to add java doc.

also update the other java doc to say "Increment item's count by one." or "Increment item's count by count"